package.json

```diff
@@ -1,38 +1,67 @@
 {
-  "name": "react_epam_training",
-  "version": "0.1.0",
-  "private": true,
+  "name": "react_webpack",
+  "version": "1.0.0",
+  "description": "",
+  "main": "webpack.config.js",
+  "scripts": {
+    "start": "webpack-dev-server --env env=development",
+    "build": "webpack --env env=production",
+    "prepare": "husky install",
+    "lint": "eslint . --fix"
```

Comment:
it's missing dev build which was required by task. Also usually real projects will have structure like `build:dev`, `build:fqa`, `build:prod` etc. Also to improve and put more structure on env files usage you can check `dotenv`.

Reply:
Change script in package.json, add build: dev and dev:prod. Add .env file